plugins/graphql-schema/src/generate.sheet.ts

```diff
@@ -9,16 +9,19 @@ export function generateSheets(
 ): Flatfile.SheetConfig[] {
   const sheets = graphQLObjects
     .map((object) => {
-      let sheetConfig =
-        sheetConfigArray?.find((config) => config.slug === object.name) || {}
+      let sheetConfig = sheetConfigArray?.find(
+        (config) => config.slug === object.name
+      )
+
+      if (sheetConfigArray.length > 0 && !sheetConfig) return
 
       const fields = object.fields
         .map((field) => generateField(field, object.name))
         .filter(Boolean)
 
       return fields.length
         ? {
-            ...sheetConfig,
+            ...(sheetConfig ?? {}),
```

Comment:
Wouldn't this always just be `...sheetConfig` if it gets past line 16 or am I reading that wrong?